azadi.core

```diff
@@ -0,0 +1,167 @@
+CAPI=2:
+name : ::azadi:1.0
+
+filesets:
+  risc:
+    depend:
+      - azadi_pkg
+      - lowrisc:ibex:ibex_core
+      #- lowrisc:tool:check_tool_requirements
```

Comment:
Remove all the commented out lines and please make sure to trim trailing white spaces from each file you modify.

Reply:
Yes, I've done this to not to lose track on the pre-build files I'm chainging but now I'll clean it up.